test/viewport.test.js

```diff
@@ -11,7 +10,55 @@ function isPhantom() {
 	return /PhantomJS/.test(navigator.userAgent);
 }
 
+describe('o-viewport utils', function () {
+	let clock;
+
+	before(function() {
+		clock = sinon.useFakeTimers();
+	});
+
+	after(function() {
+		clock.restore();
+	});
+
+	it('should throttle callback to every 100ms', function(done) {
+		const callback = sinon.spy();
+		const throttled = utils.throttle(callback, 100);
+
+		throttled();
```

Comment:
Updated now, for some reason the first time I tried that it didn't work as expected. Now it does.